src/realm/object-store/object_schema.hpp

```diff
@@ -80,7 +80,7 @@ class ObjectSchema {
     bool property_is_computed(Property const& property) const noexcept;
 
     void validate(Schema const& schema, std::vector<ObjectSchemaValidationException>& exceptions,
-                  bool for_sync) const;
+                  uint64_t validation_mode) const;
```

Comment:
Can this take a SchemaValidationMode rather than a uint64_t?

Reply:
sure :)